openmicroscopy/omero-ms-queue
# Smuggler 5.6

This brings smuggler to 5.6 gateway

To build and deploy http://c0c0n3.github.io/ome-smuggler/docs/content/examples/whirlwind-tour.html

cc @c0c0n3 

To test:

* open a terminal and checkout this PR
*  docker build -t smuggler .
* docker run -it  -p 8000:8000 smuggler 
* Make sure the user has an email address set up or use the daily build of OMERO.insight
* in ``insight/config/container.xml`` Edit ``<entry name="omero.client.import.offline.enabled" type="boolean">true</entry>``
* Import an  image using insight. Check the output in smuggler. Ignore failure when sending email. This is not set up.
